README.md

```diff
@@ -27,9 +27,50 @@ Install required packages for running deploy::
     pyvenv .
     bin/pip install -r requirements-deploy.txt
 
-Deploy
-------
+Deploy to AWS
+-------------
```

Comment:
Add the read me info that Seth put in that other PR.

Reply:
done